.github/workflows/build.yml

```diff
@@ -80,12 +80,9 @@ jobs:
       run: |
         .automation/build-rpm.sh $ARTIFACTS_DIR
 
-    - name: Create DNF repository
-      run: |
-        createrepo_c $ARTIFACTS_DIR
-
     - name: Upload artifacts
-      uses: actions/upload-artifact@v2
+      uses: ovirt/upload-rpms-action@v2
       with:
         name: rpm-${{ matrix.shortcut }}
         path: ${{ env.ARTIFACTS_DIR }}
```

Comment:
Shouldn't name and path attributes be removed?

Reply:
It should! Fixing